Gemfile

```diff
@@ -1,8 +1,15 @@
-source 'https://rubygems.org'
+source 'https://rubygems.org/'
 
-group :test do
-  gem 'rspec'
-  gem 'rubocop', '0.79.0'
-  gem 'simplecov', require: false
-  gem 'simplecov-console', require: false
+gem 'capybara'
```

Comment:
Just a small thing, but I would maintain a test group that has all related gems, such as rspec, simplecov etc.